labs/Language/Basics.md

```diff
@@ -241,25 +269,22 @@ labs::Language::Basics> ~0b000011001101 && 0o4115 || 0x0d0 ^ 9
 #### Signed versions: `<$`, `<=$`, `>$` and `>=$`
```

Comment:
It kills me that the section on signed operations doesn’t actually say anything about them